schema/system/is_multi_material.json

```diff
@@ -0,0 +1,11 @@
+
+{
+    "schemaId": "system-is-multi-material",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "is multi schema",
+    "properties": {
+        "isMultiMaterial": {
```

Comment:
why is this in `system` ? Isn't this related to `job` ?

Reply:
Per the above: let's have a followup task to re-organize the schemas and move some out of `system`. This work effort is aimed at making the webapp use ESSE schemas instead of duplicating them.